.cspell.json

```diff
@@ -0,0 +1,50 @@
+{
+    "version": "0.1",
+    "language": "en",
+    "words": [
+        "APPINSIGHTS",
+        "asar",
+        "ceapex",
+        "codecov",
+        "docascode",
+        "Docfx",
+        "docset",
+        "docsets",
+        "docsmsft",
+        "extversion",
+        "fakedcorrelation",
+        "INSTRUMENTATIONKEY",
+        "Jiayin",
+        "Keychain",
+        "Keytar",
+        "lcovonly",
+        "logcode",
+        "markdig",
+        "mimetypes",
+        "netcore",
+        "openid",
+        "openpublishing",
+        "pagejson",
+        "platformversion",
+        "postbuild",
+        "prebenchmark",
+        "Pseudonymized",
+        "remotename",
+        "Repos",
+        "revparse",
+        "Submodule",
+        "triplecrownvalidation",
+        "uikind",
+        "vscodemachineid",
+        "vscodesessionid",
+        "vscodeversion",
+        "webpack'ed"
+    ],
+    "ignorePaths": [
+        "package-lock.json",
+        "tsconfig.json",
+        "azure-pipeline.yml",
+        ".vscode-test/**",
+        "out/**"
```

Comment:
```suggestion
        "out/**",
        "coverage/**",
        "*.vsix"
```

Reply:
Good idea. I manually applied the change since I rebased